contracts/Everlong.sol

```diff
@@ -62,7 +60,7 @@ import { EverlongPositions } from "./internal/EverlongPositions.sol";
 /// @custom:disclaimer The language used in this code is for coding convenience
 ///                    only, and is not intended to, and does not, have any
 ///                    particular legal or regulatory significance.
-contract Everlong is EverlongAdmin, EverlongPositions {
+contract Everlong is EverlongBase {
```

Comment:
IMO this structure isn't as clear as the previous structure. Hyperdrive mostly follows a mixins pattern where each block of individual logic is implemented in a separate contract, and the feature contracts are combined to create the higher level interface. The nice thing about this pattern is that it's clear what the logic contracts are just from looking at the entrypoint. 

Another thought is that it's convenient to have abstract functions used by several mixins and internal functions used in several mixins in the base contract. Using this inheritance hierarchy makes that impossible and also makes the order of inheritance between the mixins more consequential.

Reply:
Have made changes in line with this to flatten things out and not have so much reliance on import ordering Let me know if there's more improvements you see! :)